src/main/java/nl/tudelft/ewi/devhub/server/database/entities/AssignedTA.java

```diff
@@ -0,0 +1,44 @@
+package nl.tudelft.ewi.devhub.server.database.entities;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import lombok.Data;
+import org.hibernate.annotations.JoinColumnOrFormula;
+import org.hibernate.annotations.JoinColumnsOrFormulas;
+import org.hibernate.annotations.JoinFormula;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+/**
+ * Created by Pilmus on 31-5-2017.
+ */
+@Data
+@Entity
+@Table(name = "assigned_ta")
```

Comment:
Could use a [`@UniqueConstraint`](http://docs.oracle.com/javaee/6/api/javax/persistence/UniqueConstraint.html) on `course_edition_id`, `group_number` and `assignment`.